test/test_profiler.py

```diff
@@ -269,6 +269,21 @@ def test_flops(self):
         print(profiler_output)
         self.assertIn("FLOPS", profiler_output)
 
+    @unittest.skipIf(not kineto_available(), "Kineto is required")
```

Comment:
pls just merge this into test_flops instead of copy pasting

Reply:
test_flops can just use `use_kineto=kineto_available()` as we have both kineto and non-kineto CI builds

Reply:
Good point!